src/hooks/useLocalNotifications.tsx

```diff
@@ -83,10 +83,21 @@ const useLocalNotificationsContext = () => {
     [setAllTransferNotifications, dydxAddress, allTransferNotifications]
   );
 
-  const addTransferNotification = useCallback(
+  const addOrUpdateTransferNotification = useCallback(
```

Comment:
change `addTransferNotification` to `addOrUpdateTransferNotification`, so that notification would override existing notifications that have the same id or txHash.